sc2reaper/sc2reaper.py

```diff
@@ -1,13 +1,15 @@
 """Main module."""
-
-import glob
 import json
+from pathlib import Path
+from typing import List
 from pymongo import MongoClient
 from pysc2 import run_configs
 from sc2reaper.sweeper import extract_all_info_once
 
-# Reading what is set up at the config.json:
-with open(str(__file__).replace('sc2reaper.py', 'config.json')) as fp:
+# Reading the set up from config.json found in the current working directory(cwd_):
+cwd_ = Path.cwd()
+config_ = (cwd_ / 'config.json')
+with config_.open() as fp:
```

Comment:
the thing about cwd is that it forces you to call the script from the `sc2reaper` folder itself.